lib/core/versions/latest/ErrorCode.ts

```diff
@@ -32,11 +32,14 @@ export default {
   OperationCreateInvalidDidDocument: 'operation_create_invalid_did_document',
   OperationCountLessThanZero: 'operation_count_less_than_zero',
   OperationExceedsMaximumSize: 'operation_exceeds_maximum_size',
-  OperationHeaderMissingKid: 'operation_header_missing_kid',
   OperationHeaderMissingOrIncorrectAlg: 'operation_header_missing_or_incorrect_alg',
-  OperationHeaderMissingOrIncorrectOperation: 'operation_header_missing_or_incorrect_operation',
-  OperationMissingOrIncorrectPayload: 'operation_missing_or_incorrect_payload',
+  OperationHeaderMissingOrIncorrectKid: 'operation_header_missing_kid',
```

Comment:
nit inconsistent name with OperationHeaderMissingOrIncorrectAlg above
```suggestion
  OperationHeaderMissingOrIncorrectKid: 'operation_header_missing_or_incorrect_kid',
```

Reply:
Thanks for observing it.